erts/emulator/beam/jit/arm/instr_common.cpp

```diff
@@ -1463,7 +1463,8 @@ void BeamModuleAssembler::emit_is_eq_exact(const ArgLabel &Fail,
     a.cmp(x.reg, y.reg);
     a.b_eq(next);
 
-    if (exact_type<BeamTypeId::Integer>(X) && exact_type<BeamTypeId::Integer>(X)) {
+    if (exact_type<BeamTypeId::Integer>(X) &&
+        exact_type<BeamTypeId::Integer>(X)) {
```

Comment:
Shouldn't this be `X` and `Y`, not `X` and `X`?

Reply:
Yes.